gst/nnstreamer/tensor_query/README.md

```diff
@@ -66,6 +70,50 @@ $ cd nnstreamer-example/bash_script/example_models
 $ ./get-model.sh ./get-model.sh object-detection-tflite
 ```
 
+### MQTT-hybrid
+Above two examples, `echo sever` and `Object-detection`, use TCP direct connection. Tensor query provides two methods for connection: TCP direct connection and MQTT-hybrid.
```

Comment:
need fixing typo in this paragraph,
valuses / connetion / transmision
Mqtt and mqtt > MQTT
tcp > TCP

Reply:
Thanks. Updated.